changelogs/unreleased/4836-Vishal-Chdry-minor.md

```diff
@@ -0,0 +1 @@
+Added support for `ALL` DNS lookup family
```

Comment:
would be good to add more context here around how this DNS lookup type will work, similar to what you've added to the CRD field description